src/dataset-builder/CohortEditor.ts

```diff
@@ -331,18 +334,18 @@ export const CriteriaGroupView: React.FC<CriteriaGroupViewProps> = (props) => {
   const [groupParticipantCount, setGroupParticipantCount] = useLoadedData<SnapshotBuilderCountResponse>();
 
   const updateGroupParticipantCount = useRef(
-    _.debounce(250, (dataset, criteriaGroup) =>
+    _.debounce(250, (snapshot, criteriaGroup) =>
```

Comment:
I think this is `snapshotId`, not `snapshot`. There's no type errors here because `_.debounce()` doesn't seem to be inferring types for its function argument. To force an error in the caller, you can add types here.
```suggestion
    _.debounce(250, (snapshotId: string, criteriaGroup: CriteriaGroup) =>
```

Reply:
Good catch, will do - thank you.